src/Pact/Gas/Table.hs

```diff
@@ -285,13 +284,14 @@ tableGasModel gasConfig =
         GPrincipal g -> fromIntegral g * _gasCostConfig_principalCost gasConfig
         GMakeList2 len msz ->
           let glen = fromIntegral len
-          in glen + maybe 0 ((* glen) . intCost) msz
+          in glen + maybe 0 ((* glen) . intCost ThresholdPrePact47) msz
         GZKArgs arg -> case arg of
           PointAdd g -> pointAddGas g
           ScalarMult g -> scalarMulGas g
           Pairing np -> pairingGas np
   in GasModel
       { gasModelName = "table"
+      , gasModelType = TableGasModel
```

Comment:
What's the difference between Table most and a Constant model? Why isn't the constant just put into the table?

Reply:
It's a distinction we already have, but I just realized it's a moot one.